### PR TITLE
Added recipe for docrep v0.1.2

### PR DESCRIPTION
**Pull request by Chilipp**
The docrep [python package](https://pypi.python.org/pypi/docrep/0.1.1) is hosted on [github](https://github.com/Chilipp/docrep) with a documentation on [readthedocs.org](http://docrep.readthedocs.io/en/latest/).
This module targets developpers that develop complex and nested Python APIs and helps them to create a well-documented piece of software.

This pull request was restarted due to the AWS outage the day before.

### CODE REVIEW

File: recipes/docrep/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
```
Comment (jakirkham) [inline recipes/docrep/meta.yaml]: Is this needed during build time? If not, could we please drop it from `build`?

**Comment by jakirkham** [comment]
Except for one comment, LGTM.

File: recipes/docrep/meta.yaml (hunk 1 above)
Comment (Chilipp) [inline recipes/docrep/meta.yaml]: No, they are not needed, sorry. Please drop them

Comment (jakirkham) [inline recipes/docrep/meta.yaml]: Feel free to let me know once you have addressed this.

File: recipes/docrep/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  imports:
+    - docrep
+
+about:
+  home: https://github.com/Chilipp/docrep
+  license: GPLv2
+  license_family: GPL2
```
Comment (jakirkham) [inline recipes/docrep/meta.yaml]: FYI: Can use `GPL` here.

**Comment by jakirkham** [comment]
Thanks @Chilipp.

**Pull request merged**